piksi_gps/piksi_gps/piksi_gps_pub.py

```diff
@@ -68,6 +74,17 @@ def __init__(self):
 
 
     def piksi_log_callback(self, signal, *args, **kwargs):
+        '''
+        This function is called when the GPS sends data over the communition medium
+        It filters the incoming message into the required datapoints and publishes them to their respective ros topics
+        Arguements
+        ----------
+        signal: the incoming message object
+        args: non keyword arguements
```

Comment:
FYI args and kwargs don't need to be documented if they go unused, should be kept in the function definition though since the callback will try to pass extra args/kwargs.